<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>-10</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 20pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Control Escolar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="numcontrolline"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nombrelineedit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Num. Control</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Semestre</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="semestreline"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Carrera</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="carreraline"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>440</y>
      <width>701</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="botonactua">
       <property name="text">
        <string>Actualizar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/act.png</normaloff>../img/act.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botoninsert">
       <property name="text">
        <string>Insertar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/ins.png</normaloff>../img/ins.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotondatosPersonales">
       <property name="text">
        <string>Datos Personales</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/dts.png</normaloff>../img/dts.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonMaterias">
       <property name="text">
        <string>Materias</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/mts.png</normaloff>../img/mts.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonCursan">
       <property name="text">
        <string>Cursan</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/curs.png</normaloff>../img/curs.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonbuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/buscar.png</normaloff>../img/buscar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botoneliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/eleminar.png</normaloff>../img/eleminar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="botonhistorial">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Historial Academico</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/dtbs.png</normaloff>../img/dtbs.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Creditos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="creditoLine"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="botonsalir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
